en/docs/release-notes/version-3.3.0.md

```diff
@@ -1,15 +1,15 @@
 # Ballerina VSCode extension (version 3.3.0) 
 
-We are happy to announce the Ballerina VS Code plugin 3.3.0 release, which has some exciting new features. Following are the highlights of this release.
+We are happy to announce the Ballerina VS Code extension 3.3.0 release, which has some exciting new features. Following are the highlights of this release.
 
 - [Visual Data Mapper](#data-mapper) - Helps you write and visualize data transformations easily
-- [GraphQL Tryit](#integrated-graphql-tryit) - Facilitates trying out the GraphQL services with the integrated client 
-- [Improved record editor](#record-editor) - Provides a better editing experience with suggestions 
-- [Project Design View](#record-editor) - Facilitates visualizing service interactions in your project
+- [Integrated GraphQL Tryit](#integrated-graphql-tryit) - Facilitates trying out the GraphQL services with the integrated client 
+- [Improved Record Editor](#record-editor) - Provides a better editing experience with suggestions 
```

Comment:
Shouldn't we use sentence case?

Reply:
UI elements (such as  the Record Editor) can be special cased as pronouns and should be in title case.

Reply:
Yes, valid for features like Data Mapper Record Editor etc.